src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -469,6 +469,7 @@ Copyright (c) .NET Foundation. All rights reserved.
                             ILLink;
                             CreateReadyToRunImages;
                             GeneratePublishDependencyFile;
+                            PrepareForPublish;
```

Comment:
I'd add PrepareForPublish as the first dependency -- there were a few other uses of `$(PublishDir)`, so just in case.

Reply:
Done